tests/python/tests/conftest.py

```diff
@@ -504,6 +511,7 @@ def spark(warehouse: Warehouse, storage_config):
         f"spark.sql.catalog.{catalog_name}.scope": settings.openid_scope,
         f"spark.sql.catalog.{catalog_name}.oauth2-server-uri": f"{settings.token_endpoint}",
     }
+    print(f"Configuration: {configuration}")
```

Comment:
could there be credentials in configuration?

Reply:
There could be - shouldn't be here